code/modules/projectiles/guns/energy/energy_gun.dm

```diff
@@ -82,7 +84,10 @@
 /obj/item/gun/energy/e_gun/dragnet/snare
 	name = "Energy Snare Launcher"
 	desc = "Fires an energy snare that slows the target down."
+	w_class = WEIGHT_CLASS_HUGE
```

Comment:
The snare launcher doesn't even use an internal cell, it wasn't used a whole lot I don't see why it should need a weight class nerf